src/python/AsyncStageOut/PublisherWorker.py

```diff
@@ -836,8 +835,12 @@ def format_file_3(self, file):
              }
         file_lumi_list = []
         for run, lumis in file['runlumi'].items():
-            for lumi in lumis:
-                file_lumi_list.append({'lumi_section_num': int(lumi), 'run_num': int(run)})
+            for lumi, evts in lumis.items():
+                self.logger.debug("lumi: %s, evts: %s" % (lumi,evts))
+                if not evts in ['None']:
```

Comment:
You're right, during the test there were 2 values in the list that's the reason. Now it makes no sense other than be easier to add another value ^^